src/applications/coronavirus-vaccination/hooks/useSubmitForm.jsx

```diff
@@ -1,30 +1,28 @@
 import { useState, useCallback } from 'react';
 
 import environment from 'platform/utilities/environment';
-import { apiRequest } from 'platform/utilities/api';
+import { apiPostRequest } from '../apiCalls';
 import { requestStates } from 'platform/utilities/constants';
 
 const apiUrl = `${environment.API_URL}/covid_vaccine/v0/registration`;
+const unauthenticatedApiUrl = `${apiUrl}/unauthenticated`;
 
 export default function useSubmitForm() {
   const [status, setSubmitStatus] = useState(requestStates.notCalled);
 
   const submit = useCallback(formData => {
-    async function sendToApi() {
-      try {
-        await apiRequest(apiUrl, {
-          method: 'POST',
-          body: JSON.stringify(formData),
-          headers: {
-            'Content-Type': 'application/json',
-          },
-        });
+    const sendToApi = async () => {
+      const submissionUrl = formData.isIdentityVerified
+        ? apiUrl
+        : unauthenticatedApiUrl;
 
+      try {
+        await apiPostRequest(submissionUrl, formData);
         setSubmitStatus(requestStates.succeeded);
       } catch (error) {
         setSubmitStatus(requestStates.failed);
       }
-    }
+    };
```

Comment:
Not sure, but I think your merge resolution may have gone wrong because `${apiUrl}/unauthenticated` route isn't used anymore

Reply:
Thanks Nick. I'll update that now.